### PR TITLE
Tiny code refactoring

### PR DESCRIPTION
**Pull request by mrwrob**

### CODE REVIEW

File: graph_api/node/node_service.py [hunk 1]
```diff
@@ -16,7 +18,7 @@ class NodeService:
         .replace("{database_name}", database["name"])
     database_auth = HTTPBasicAuth(database["user"], database["passwd"])
 
-    def save_node(self, node):
+    def save_node(self, node: NodeIn):
```
Comment (mrwrob) [inline graph_api/node/node_service.py:21]: If we've got appropriate class, why not to use it...

File: graph_api/node/node_service.py [hunk 2]
```diff
@@ -26,21 +28,22 @@ def save_node(self, node):
         Returns:
             Result of request as node object
         """
-        create_statement = "CREATE (n) RETURN n"
-        for label in node.labels:
-            create_statement = create_statement[:-10] + f":{label}" + create_statement[-10:]
+        create_template = string.Template("CREATE (n:$labels) RETURN n")
```
Comment (mrwrob) [inline graph_api/node/node_service.py:31]: string.Template seems to be more suitable for inserting values inside the string. "Guessing" the position where to put value isn't a good idea ;)

File: graph_api/node/node_service.py [hunk 3]
```diff
@@ -26,21 +28,22 @@ def save_node(self, node):
...
-        result = requests.post(url=self.database_url,
-                               json=commit_body,
-                               auth=self.database_auth).json()
-        if len(result["errors"]) > 0:
-            result = NodeOut(errors=result["errors"])
+        response = requests.post(url=self.database_url,
+                                 json=commit_body,
+                                 auth=self.database_auth).json()
+        if len(response["errors"]) > 0:
+            result = NodeOut(errors=response["errors"])
```
Comment (mrwrob) [inline graph_api/node/node_service.py:44]: It's never a good idea to mutate the object or change the value of variable based on itself. Here we had one `result` that was the response from Neo4J request, and another `result` that was a returned value from function. They're not the same, thus it's better to have different names - i.e. `result` and `response`.

**Review by RioluM — APPROVED** [review]
Valid comments, we will keep to them

**Pull request merged**